src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/EditTypePart.cshtml

```diff
@@ -43,6 +43,13 @@
 
     @if (Model.Shape.Content != null)
     {
+        @if (Model.Shape.Shortcuts != null)
+        {
+            <div class="form-group">
+                @await DisplayAsync(Model.Shape.Shortcuts)
```

Comment:
Just noting a new zone `Shortcuts` - and you have moved all the existing create templates to the zone as well.

Did we need a new zone? was positioning not enough?

But looks ok

Reply:
Positioning stacks shortcuts vertically, and there are many things that just use "Content" as position. So shortcuts could be mixed with other editors.
The new zone ensures shortcuts are at the top and displayed inline.